invenio/base/scripts/apache.py

```diff
@@ -19,11 +19,57 @@
 
 from __future__ import print_function
 
+import os
+import socket
+
 from invenio.ext.script import Manager, change_command_name
 
 manager = Manager(usage="Perform Apache operations.")
 
 
+def _detect_ip_address():
```

Comment:
Take a look at lines 82 and 117 in this file where this and the next function are imported from `invenio.legacy.inveniocfg`.